src/providers/krb5/krb5_child.c

```diff
@@ -1981,7 +1981,7 @@ static errno_t tgt_req_child(struct krb5_req *kr)
     kerr = krb5_get_init_creds_password(kr->ctx, kr->creds, kr->princ_orig,
                                         password_or_responder(password),
                                         sss_krb5_prompter, kr, 0,
-                                        SSSD_KRB5_CHANGEPW_PRINCIPAL,
+                                        discard_const(SSSD_KRB5_CHANGEPW_PRINCIPAL),
```

Comment:
It was agreen in #50 that this patch will not be included
https://github.com/SSSD/sssd/pull/50#discussion_r87217214